apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -118,6 +119,10 @@
             <version>${ststem-rules.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
```

Comment:
Don't add a dependency, we need to relocate all class package names in the agent. Unless it is necessary, we want the agent dependency lib as less as possible.